pom.xml

```diff
@@ -75,6 +75,7 @@
         <maven.checkstyle.plugin.version>3.1.2</maven.checkstyle.plugin.version>
         <maven.compile.version>3.10.1</maven.compile.version>
         <maven.compiler.release>11</maven.compiler.release>
+        <maven.compiler.source>11</maven.compiler.source>
```

Comment:
I don't understand. I'm pretty sure if `maven.compiler.release` is set, m-compiler-p ignores completely `.source` (a `.target`).
Why it was classified as _missing_?